azure-pipelines.yml

```diff
@@ -0,0 +1,30 @@
+# Starter pipeline
+
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: Npm@1
+  inputs:
+    command: 'custom'
+    customCommand: --no-git-tag-version version prerelease --preid=$(Build.BuildNumber)
+    workingDir: 'Source/MSBuildTools.Unity/Assets/'
+  displayName: 'Update UPM Version'
+  
+- task: Npm@1
+  inputs:
+    command: 'publish'
+    workingDir: 'Source/MSBuildTools.Unity/Assets/'
+    publishRegistry: 'useFeed'
+    publishFeed: '0cb95e25-9194-4ccd-9afb-439b25ecb93a/a3d1c3cc-6042-4e05-b699-39a947e75639'
```

Comment:
>'0cb95e25-9194-4ccd-9afb-439b25ecb93a/a3d1c3cc-6042-4e05-b699-39a947e75639' [](start = 17, length = 75)

Where does this come from? I thought we'd use the NPM registry url listed in the Artifacts section.

Reply:
Thats the feed id of that registry, it was added automatically.

Reply:
Oh, I thought this was a hand authored file.

---
In reply to: [314902182](https://github.com/microsoft/MSBuildForUnity/pull/11#discussion_r314902182) [](ancestors = 314902182)